cmd/root.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "obscli",
+	Short: "Command-line interface for managing OpenBuildService (OBS) assets",
+	Long: `obscli is a Comman-line interface that is used to manage the Kubernetes packages
+	hosted on the OpenBuildService (OBS) Platform.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("root command function called")
+	},
```

Comment:
You can do something like this instead:
```suggestion
	RunE: func(cmd *cobra.Command, _ []string) {
		return cmd.Help()
	},
```